marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -191,9 +194,21 @@ handleSubRoute _ Router.ActusBlocklyEditor = selectView ActusBlocklyEditor
 
 handleSubRoute _ Router.Wallets = selectView WalletEmulator
 
+-- This route is supposed to be called by the github oauth flow after a succesful login flow
+-- It is supposed to be run inside a popup window
+handleSubRoute settings Router.GithubAuthCallback = do
+  authResult <- runAjax $ runReaderT Server.getApiOauthStatus settings
+  case authResult of
+    (Success authStatus) -> liftEffect $ informParentAndClose $ view authStatusAuthRole authStatus
+    -- TODO: is it worth showing a particular view for Failure, NotAsked and Loading?
+    -- Modifying this will mean to also modify the render function in the mainframe to be able to draw without
+    -- the headers/footers as this is supposed to be a dialog/popup
```

Comment:
I think not until we get user feedback or analytics that suggest that it is happening. I would change this `TODO` to a normal comment suggesting that we could.